module/VuFind/src/VuFind/RecordTab/PluginManager.php

```diff
@@ -74,6 +75,7 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
         CollectionList::class => CollectionListFactory::class,
         Description::class => InvokableFactory::class,
         Excerpt::class => ExcerptFactory::class,
+        Formats::class => InvokableFactory::class,
```

Comment:
You're using InvokableFactory here, so can you delete the FormatsFactory you defined above? Also, it's probably worth adding an alias `'formats' => Formats::class` for completeness. (There are some defaults that probably auto-guess all of this and make it work, but I try to define all the configs explicitly to clarify what is available).

Reply:
Again, I'm pretty sure I understood this and did it. I removed the factory and added the alias.